### ISSUE TITLE
Filtering items through servo with same IDs and metadata:0 does not work

### ISSUE THREAD

**Issue by crogers923** [CoFH/Feedback#1394]
Using 1.12.2-2.5.1.14 jar in Direwolf 20. Trying to use a servo (have tried both regular and reinforced) to filter Thaumcraft's [vis crystals](https://ftb.gamepedia.com/Vis_Crystal). It will not filter these items. If I shift+click one type into the servo/filter, it won't let me shift+click others into it. I can still manually drag the others. They have the same ID but different NBT

information for the crystals from advanced tooltips:
`thaumcraft:crystal_essence`
`Metadata:0`
`NBT: {Aspects:[{amount:1,key:"aqua"}]}`

NBT keys vary between `aqua`, `ordo`, `terra` , `aer`, `ignis`, and `perditio`. None will come through if any are set on the filter for whitelist. I've tried every combination of filtering without success. As soon as one of these crystals is put in on the white list, none of them are pulled.

**Comment by Tora-B** [comment]
First, it seems like you have whitelists and blacklists backwards. Second, are you aware of the NBT filtering option, available on Reinforced tier and above?

**Comment by crogers923** [comment]
I do not have whitelist and blacklist backwards, but maybe I explained poorly. I'm using white lists with only these crystals, and they are not going through. I have tried using nbt, and it is not working.

I have one reinforced servo on a box with all 6 of the crystals that is active so it pulls. I have a reinforced filter on another box with a white list of all 6 types of crystals. They do not leave the first box. If I remove the filter, they pull and go to the box without issue.

I've also tried an OFF reinforced servo to act as the filter with no success, just to cover my bases. 

I have this same setup across various boxes for ~100 different items. These are the only ones causing me issues.

None of these 6 crystals will pull if they are whitelisted.

If it'll help, I can send screen shots when I'm home
